### ISSUE TITLE
Checking 1 min data

### ISSUE THREAD

**Issue by teraita** [INTERMAGNET/wg-definitive-data#2]
Roman started discussion of Checking 1s data. It looks nice. I think 1 min minute checking needs something similar and the checklist can be written here. We have discussion document DD.5 open in NextCloud, where comments /ideas written now.

**Comment by leonro** [comment]
> Could the data checking process be followed in Github? Then checker can pull request to community in the problematic case. Also the delays are then followed.How others see this?

Yes, this is possible. You can add a simple table with results as  I did in a very preliminary example for one-second. You can also rise an individual issue for checking a specific Observatory e.g. WIC2016_step1, describe the issue and assign it other data checker which you would have their opinion. They automatically will get an e-mail notfication and can comment. The discussion is open and transparent. You can add graphs, links, whatsoever. If all aspects are solved you close the issue and thats it.

**Comment by benoitheumez** [comment]
Tero, Is it here the place to upload the guidelines for datacheckers?

**Comment by teraita** [comment]
I did the wd-dd-datachecking, where I export issues from the document draft. It is still private, you have access. This section seems to be public, open to anyone?

**Comment by benoitheumez** [comment]
Here is the reviewed guide for datacheckers (PDF)

[Guidance_how_to_check_1min_v1.0.pdf](https://github.com/user-attachments/files/21662498/Guidance_how_to_check_1min_v1.0.pdf)

.
<html><head></head><body>It has been condensed to 4 pages and 2 extra for a
              checklist for ease to understand and follow. It
              includes all the necessary information to standardise
              data-checking.<br>
              It includes the role of datacheckers, description of the
              datachecking process, states the must-used tools (without
              details on how to run/use) and review IM criteria and
              standards.<br>
              We have added text about "permission for data checkers to
              modify text files on STEP1" and "Absolute pillar F is
              reported frequently" in the checklist to comply with DD6
              and DD.7:<br>
              <br>
              
DD.6 | DataCheckers, BH, TR | Clarify permission for data checkers to modify                       files, to be added into the data checkers                       guidelines.                                              No changes to be made on existing files in step1.                       Give 4 weeks to imos to make change before it is                       made automatic |  
-- | -- | -- | --
DD.7 | TR, BH | Frequency of absolute S measures. Data checkers                       to add this parameter on check list.                                              And communicate with IMOs to either report missing                       values or start performing regular measures. Only                       for recent data. | TR volunteered to make a script to detect gaps                       or lack of pilar measure

</body></html>
